### DISCUSSION TITLE
Upgrading Python

### DISCUSSION THREAD

**Discussion by cyberphor** [Security-Onion-Solutions/securityonion#8640] [General] [answered]
Question: is there a date scheduled for upgrading Python to the latest version? 

From what I understand most of the containers are running a more current version, but the underlying OS is not.

**Comment by dougburks** [comment, marked as answer]
Security Onion 2.4 will have a newer base OS which includes newer Python. We plan to announce more details at Security Onion Conference:
https://securityonionsolutions.com/conference/

**Comment by cyberphor** [comment]
Thanks @dougburks!
